jiant/utils/tokenizers.py

```diff
@@ -58,6 +58,17 @@ def tokenize(self, sentence):
         return self.lookup_ids(ids)
 
 
+class SplitCharsTokenizer(Tokenizer):
```

Comment:
add a quick comment on how this works ?
So `tokenize` should get a string (a word) and it'll split into characters ?